app/views/tasks/_form.html.haml

```diff
@@ -49,4 +49,4 @@
         .center-block
           %button.btn.btn-default{type: 'button', data: {dismiss: 'modal'}}
             = t(:actions)[:close]
-          %button.btn.btn-primary{type: 'submit'}=t(:actions)[:save]
+          %button#new_task_submit.btn.btn-primary{type: 'submit'}=t(:actions)[:save]
```

Comment:
I think we can change this to a `submit_tag` rather than a button and Rails will automatically disable on submit

Reply:
Well dang, you are in fact correct. Much cleaner!